kore.maude

```diff
@@ -220,18 +227,30 @@ Questions about this definition:
 - do we really need ~> as a K construct?  Can it be a label and thus the way K
   deal with evaluation contexts just a particular methodology and not an
   intrisic part of K ? Same question for =>.
+  Cosmin: Syntacticly at least, I would say "no" for both. From the implementation
```

Comment:
From an implementation perspective, I do want to modify the KORE data structure to allow for traversal of `=>` and `~>` as regular terms -- it makes lots of code smaller. Semantically, and in the K definition of KORE, I have no strong preference. I am also inclined to leave `=>` as is. `~>` on the other hand could be made into a regular assoc list easily -- from what I remember, it already is in the latest Java backend implementation.